### ISSUE TITLE
[BUG] Tardis level points

### ISSUE THREAD

**Issue by Soyaro666** [DarkholmeTenk/TardisModIssues#183]
I'm not able to invest unused level points

Didn't pay attention to them for a while. now I've collected 2 and wanted to invest them but the buttons don't react to sneak+rightclick any more

**Comment by DarkholmeTenk** [comment]
It's right click + sneak right click iirc

**Comment by Soyaro666** [comment]
I know. The Tardis tells me when I rightclick the button. I did it multiple times before. It didn't work, thats because I postet this. I'll check again after I installed any updates you possibly made while I wasn't at home. Give me a few hours...
edit: okay, no new builds since I updated last. I'll test now

**Comment by PrincessOfEvil** [comment]
You need to be very quick. It only registers shiftclicks while it's down.

2015-06-01 20:17 GMT+06:00 Soyaro666 notifications@github.com:

**Comment by Soyaro666** [comment]
whatever my tardis had in that moment. I tried it multiple times (10 to 20 I guess) and it didn't work. now it worked on the first try...
could this Problem result from too less energy? I'm in the End (costs 5000 energy) but had less than 3000 (thats why I wanted to raise energy generation), now, while it worked, I have a little more than 5000... just a theory of course, but I want to find out how to reproduce my problem...

@laz2727 yep, I know. Like I said, I did it multiple times before. This Tardis is level 23 now and I only had 2 unspent points so it worked at least 9 times before (and that's only my survival savegame of course)
